### ISSUE TITLE
Live updating

### ISSUE THREAD

**Issue by bnomei** [CalebGrove/select-a-structure#3]
question: from looking at the gif... it needs a 'save' before the dropdown has the new values of the structure, right?

**Comment by calebgrove** [comment]
Yep. Of course, this is only noticeable when the select is on the same page as the structure.

In an ideal world, the select field would use use Javascript when it's on the same page as the structure and build the list of options "live" when you clicked on it. But, honestly, I'm no good (read: *really slow*) at JS and don't have the time.

I'll leave this open for future reference, though, as this would be a nice change and maybe someone would create a PR for it. :)
